libs/shared/create-course-form/src/lib/shared-create-course-form.scss

```diff
@@ -1,23 +1,30 @@
 $error: #3f51b5;
 
-.inner-container {
-  margin-top: 1rem;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  flex-direction: column;
-  .sub-header {
-    text-align: center;
+.course-form-container {
+  padding: 30px;
+  .header {
     font-size: 18px;
-    margin-bottom: 7px;
+    margin-bottom: 8px;
   }
-
   .draggable-container.empty {
     min-height: 7em;
   }
+
+  .lessonsLabel,
+  .categoriesLabel {
+    padding-left: 17px;
+  }
+
+  .MuiSelect-selectMenu {
+    white-space: normal !important;
```

Comment:
I would prefer to avoid !important if possible. Write full class/elements path instead.